pkgs/development/python-modules/pxml/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, pythonAtLeast
+, isPy27
+, buildPythonPackage
+, fetchPypi
+, blessings
+, six
+, nose
+, coverage
+}:
+
+buildPythonPackage rec {
+  pname = "pxml";
+  version = "0.2.13";
+  disabled = pythonAtLeast "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0c9zzfv6ciyf9qm7556wil45xxgykg1cj8isp1b88gimwcb2hxg4";
+  };
+
+  propagatedBuildInputs = [ blessings six ];
+  checkInputs = [ nose coverage ];
+
+  checkPhase = ''
+    nosetests -e 'test_prefixedWhitespace'
```

Comment:
can you add a comment why this test is excluded?

Reply:
/status needs_work

Reply:
done.

/status needs_review